components/layout_thread/lib.rs

```diff
@@ -954,6 +954,21 @@ impl LayoutThread {
             }
             let display_list = (*rw_data.display_list.as_ref().unwrap()).clone();
 
+            for item in &display_list.list {
+                match *item {
+                    DisplayItem::Iframe(ref iframe) => {
+                        let size = Size2D::new((*item).bounds().size.width.to_f32_px(),
+                                               (*item).bounds().size.height.to_f32_px());
+                        debug!("new iframe size: {:?}", size);
```

Comment:
Rather than iterating over the complete display list every time we build one, we should send this message when we create an iframe display item instead: https://dxr.mozilla.org/servo/rev/1f76aa6ef7ec7db0a0c40223874a72e8f4059deb/components/layout/display_list_builder.rs#1441